WordPress/gutenberg
# Improve text and design of the block removal warnings

## What?

It improves the text and design of the block removal warnings:

* When the user is trying to delete only one block, add the whole text in one line.
* Change the generic message from "Removing this block is not advised." to something more open, like "Only proceed to remove this block if you're absolutely sure of what you're doing."

## Why?

The goal is to improve the UX and have a clearer message/copy.

Closes #52392.

## How?

I checked how this was added technically, and "Removing this block is not advised." is added by default on all block removal warning models.

I believe that part of the reason that it was added in this more "generic" way is to be able to loop and show a list with all warnings in case the user tries to delete multiple blocks:

With that in mind, I kept a similar structure but added this text directly to its singular/plural versions.

## Testing Instructions

1. Select the "protected" block that you would like to delete ("Query Loop" block would be an example of that).
2. Click on the Delete key to see the modal — you can compare the different versions.

## Screenshots or screencast 

**One warning:**

* Before (previous text and different lines) 
![image](https://github.com/WordPress/gutenberg/assets/40555737/680642d7-2b16-4dc0-9e96-27d7854689a9)
* After (new text and using the same line)
![image](https://github.com/WordPress/gutenberg/assets/40555737/862793f5-4031-44c6-a5d2-79d3c4594cec)

**Two or more warnings:**

* Before (previous text) 
![image](https://github.com/WordPress/gutenberg/assets/40555737/be1638fd-30af-41c9-9328-aa36ebbbebcd)
* After (new text)
![image](https://github.com/WordPress/gutenberg/assets/40555737/3b1c6895-5348-4338-a03f-0ee23963077d)
